src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/AngularDistribution/Builder/src/isotropic.hpp

```diff
@@ -0,0 +1,4 @@
+Builder& isotropic() {
+  this->representations_.emplace_back( Isotropic{} );
+  return *this;
```

Comment:
There are some places where I don't `add()` because it is just a simple function that is invoked rather than a builder. Another place I do it is when adding a *Q*-value to a reaction [here](https://github.com/njoy/ACEtk/blob/feature/CENBuilder/src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Reaction/Builder/src/Q.hpp).